lib/Conversion/StandardToHandshake/StandardToHandshake.cpp

```diff
@@ -1066,18 +1075,17 @@ LogicalResult LoopNetworkRewriter::processOuterLoop(Location loc,
            << "Multiple exits detected within a loop. Loop task pipelining is "
               "only supported for loops with unified loop exit blocks.";
 
-  BackedgeBuilder bebuilder(*rewriter, loopInfo.loopHeader->front().getLoc());
+  BackedgeBuilder bebuilder(*rewriter, loop->getHeader()->front().getLoc());
```

Comment:
nit: maybe cache the result of `getHeader` inside a variable to avoid repeated method calls?